src/PoshGitTypes.ps1

```diff
@@ -273,6 +273,8 @@ class PoshGitPromptSettings {
     [string]$DescribeStyle = ''
     [psobject]$WindowTitle = {param($GitStatus, [bool]$IsAdmin) "$(if ($IsAdmin) {'Admin: '})$(if ($GitStatus) {"$($GitStatus.RepoName) [$($GitStatus.Branch)]"} else {Get-PromptPath}) ~ PowerShell $($PSVersionTable.PSVersion) $([IntPtr]::Size * 8)-bit ($PID)"}
 
+    [bool]$TabTitle = $true
```

Comment:
I think we should consider following the pattern that `$WindowTitle` uses.  By specifying a scriptblock, users will be able to easily customize the text that gets put in the tab title.  To simplify the scriptblock, you can use functions.  Perhaps a `Get-UniqueTabTitle` to handle the logic for appending a number to the title to make it unique.  Perhaps it would be something like this:
```
[psobject]$TabTitle = {param($GitStatus, [bool]$IsAdmin) "if ($GitStatus) { Get-UniqueTabTitle "$($GitStatus.RepoName) [$($GitStatus.Branch)]" }}
```